src/pocketmine/Player.php

```diff
@@ -2864,6 +2864,10 @@ public function handleDropItem(DropItemPacket $packet) : bool{
 			// Windows 10 Edition drops the contents of the crafting grid on container close - including air.
 			return true;
 		}
+		
+		if($this->gamemode === self::SPECTATOR) {
```

Comment:
The code can be read just fine, if you have issues reading it I would suggest visiting the eye doctor. Nor does the code make developing harder, if 14 extra chars on a line make things considerably  difficult for you thats not my issue.